crates/bootstrap_srv/src/http.rs

```diff
@@ -111,40 +118,78 @@ struct Ready {
     shutdown: axum_server::Handle,
 }
 
+#[derive(Clone)]
+pub struct AppState {
+    pub h_send: HSend,
+    pub sbd_state: Option<crate::sbd::SbdState>,
+}
+
 type BoxFut<'a, T> =
     std::pin::Pin<Box<dyn std::future::Future<Output = T> + Send + 'a>>;
 
 fn tokio_thread(
-    config: ServerConfig,
+    config: Arc<Config>,
+    server_config: ServerConfig,
     ready: tokio::sync::oneshot::Sender<std::io::Result<Ready>>,
 ) {
+    tracing::trace!(?config, "Starting tokio thread");
+
     tokio::runtime::Builder::new_multi_thread()
         .enable_all()
         .build()
         .unwrap()
         .block_on(async move {
             let (h_send, h_recv) =
-                async_channel::bounded(config.worker_thread_count);
+                async_channel::bounded(server_config.worker_thread_count);
 
-            let app: Router = Router::new()
+            let sbd_config = Arc::new(config.sbd.clone());
+
+            let ip_rate = Arc::new(sbd_server::IpRate::new(sbd_config.clone()));
+
+            let c_slot = if !config.no_sbd {
+                Some(sbd_server::CSlot::new(sbd_config.clone(), ip_rate.clone()))
+            } else {
+                None
+            };
```

Comment:
Double negatives are confusing : )

```suggestion
            let c_slot = if config.no_sbd {
                None
            } else {
                Some(sbd_server::CSlot::new(sbd_config.clone(), ip_rate.clone()))
            };
```